src/Response.php

```diff
@@ -25,7 +24,7 @@ class Response implements ReadableStreamInterface
     private $headers;
     private $readable = true;
 
-    public function __construct(DuplexStreamInterface $stream, $protocol, $version, $code, $reasonPhrase, $headers)
+    public function __construct(ReadableStreamInterface $stream, $protocol, $version, $code, $reasonPhrase, $headers)
```

Comment:
:+1: 

For the reference only: This change preserves BC because you can still pass in a `DuplexStreamInterface` because it also implements the `ReadableStreamInterface`.